### ISSUE TITLE
openstack/loadbalancer/v2/loadbalancers lack of `Time.time CreatedAt/UpdatedAt` fields

### ISSUE THREAD

**Issue by naumvd95** [gophercloud/gophercloud#1680]
Need to add Time.time Created/Updated fields inside results.go, mapped from json:"updated_at"/json:"created_at"

General API reference:https://docs.openstack.org/api-ref/load-balancer/v2/index.html?expanded=show-load-balancer-details-detail#show-load-balancer-details

that fields defined in:
https://github.com/openstack/octavia/blob/22df2e0f484b3560cbb47d33b7f2f62cb57519fd/octavia/api/v2/types/load_balancer.py#L46

**Comment by naumvd95** [comment]
will do by myself

**Comment by naumvd95** [comment]
faced with issue during PR checking, time format from octavia response seems slightly differs and cannot be parsed by sane-default time.Time configuration
```
parsing time ""2019-07-15T15:57:42"" as ""2006-01-02T15:04:05Z07:00"": cannot parse """ as "Z07:00"
```

**Comment by naumvd95** [comment]
related go issue https://github.com/golang/go/issues/21990

**Comment by jtopjian** [comment]
@naumvd95 Please see https://github.com/gophercloud/gophercloud/pull/1671

**Comment by naumvd95** [comment]
@jtopjian  Awesome !! 
so I'll refactor my PR and add similar custom struct

**Comment by naumvd95** [comment]
@jtopjian btw, seems  in fixtures u left some `Z` cases like in networks
https://github.com/gophercloud/gophercloud/pull/1671/files#diff-1e0b8676b22caba7b7d81baf122157daR198
its ok?

**Comment by jtopjian** [comment]
Yes, per #1670, #1671 is meant to support _both_ timezone formats. In similar manner, you'll want to include fixtures for both formats.

**Comment by naumvd95** [comment]
@jtopjian ready for review

**Comment by jtopjian** [comment]
Implemented in #1681
